app/src/main/java/com/github/amusinamaria/ui/details/DetailsFragment.kt

```diff
@@ -4,38 +4,46 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.ImageView
 import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
-import com.github.amusinamaria.R
-import com.github.amusinamaria.repository.MockDataSource
+import com.github.amusinamaria.databinding.FragmentDetailsBinding
+import com.github.amusinamaria.repository.MovieCard
+import com.github.amusinamaria.ui.MainActivity.Companion.MOVIE_ARGS_KEY
 
 class DetailsFragment : Fragment() {
 
-    private lateinit var castRecycler: RecyclerView
+    private var _binding: FragmentDetailsBinding? = null
```

Comment:
In onDestroyView, you must set this field to null.

Reply:
Done